dll/cpl/sysdm/virtmem.c

```diff
@@ -569,27 +577,45 @@ OnSelChange(HWND hwndDlg, PVIRTMEM pVirtMem)
         }
 
         /* Set minimum pagefile size */
-        SetDlgItemText(hwndDlg, IDC_MINIMUM, _T("16 MB"));
+        SetDlgItemText(hwndDlg, IDC_MINIMUM, _T("2 MB"));
```

Comment:
In your description you list 'hardcoded strings that need to be localized ("MB") but this is still not localizable.

Reply:
Will be fixed in a separate ticket PR later https://jira.reactos.org/browse/CORE-16915
Description updated (good point)